pjsip-apps/src/pjsua/pjsua_app_common.h

```diff
@@ -91,6 +91,14 @@ typedef struct pjsua_app_config
     unsigned                buddy_cnt;
     pjsua_buddy_config      buddy_cfg[PJSUA_MAX_BUDDIES];
 
+    unsigned                inv_hdr_cnt;
+    pj_str_t                inv_hname_cfg[16];
+    pj_str_t                inv_hvalue_cfg[16];
```

Comment:
Here an above: please don't use magic numbers. Where does 16 come from?

Reply:
I've added a define PJSUA_MAX_CUSTOM_HEADERS to pjsua.h